src/Documentation/sidebar.json

```diff
@@ -120,6 +120,80 @@
         "label": "Managing External Data",
         "slug": "managing-external-data"
       },
+      {
+        "label": "Remotes",
```

Comment:
Come up with a better name - Setting up storage?

Move the section up.

Reply:
> Come up with a better name - Setting up storage?

I am renaming to "DVC Remotes". I am using "DVC Storage" for the mirror of `.dvc/cache/`. A "DVC Remote" is analogous to a Git Remote, and is broader than a "DVC Storage" (which is just a mirror/backup of the local cache). This terminology is just a convention (for example instead of "DVC Storage" we could have used "DVC Cache Backup".

Reply:
Please don't resolve comments that are not created by you. As I mentioned already, it's extremely hard to follow and track changes this way.

I would prefer to have something human readable . Otherwise we are defining remote through remotes. `Remote is remote ...` kind-a of thing.

Reply:
> I would prefer to have something human readable . Otherwise we are defining remote through remotes. Remote is remote ... kind-a of thing.

I don't understand this. We are talking about the remotes that are managed with the command `dvc remote`. What could be a better name than "Remote" or "DVC Remote"?